### PR TITLE
Update start, end, core and monument messages

### PR DESCRIPTION
**Pull request by Pablete1234**
Updates start and end messages, and adds titles
Updates core and monument message

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java [hunk 1]
```diff
@@ -31,25 +31,17 @@ public void unload() {
...
             } else {
-                ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.DARK_PURPLE + "# # {0}" + ChatColor.DARK_PURPLE + " # #", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getCompleteName())));
+                ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage("{0}", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getCompleteName() + ChatColor.WHITE)));
             }
-        }
-        ChatUtil.getGlobalChannel().sendMessage(ChatColor.DARK_PURPLE + "# # # # # # # # # # # #");
-    }
+        } else {
+        ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage("{0}", new LocalizedChatMessage(ChatConstant.UI_MATCH_OVER)));
+		}
```
Comment (iEli2tyree011) [inline src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java]: Not sure what happened here, but you should fix this bracket formatting.

**Comment by twizmwazin** [comment]
When you fix elly's issue, amend the commit rather than opening a third PR.

**Comment by Pablete1234** [comment]
i know, i know, i just didn't know how to merge the branch, that's why i had to open it again

**Comment by Pablete1234** [comment]
I added the "your team won" and "your team lost" messages

**Comment by Alfalik** [comment]
From testing, I didn't get any error, and the "Your team won" and "Your team lost" works how they should. Looks pretty good.

**Comment by Pablete1234** [comment]
I am trying to add titles, @twizmwazin said someone else to use : [https://github.com/OvercastNetwork/SportBukkit/blob/master/Bukkit/0034-Title-API.patch] , but what's the include needed for that / how can i get it?

**Comment by twizmwazin** [comment]
@Pablete1234 It is just player.showTitle(BaseComponent title)

**Comment by Pablete1234** [comment]
@twizmwazin do I not need to add any imports?

**Comment by twizmwazin** [comment]
@Pablete1234 You probably will have to for BaseComponents and TextComponents, but your IDE should do that automatically for you.

**Comment by Pablete1234** [comment]
@twizmwazin how can i localize the title? if i first create the base component i can't tell what lang it should be in, and I can't create it inside the player.showTitle()

**Comment by Pablete1234** [comment]
I'm trying to do:
```BaseComponent[] subtitle = ComponentSerializer.parse(new LocalizedChatMessage(ChatConstant.UI_MATCH_TEAM_WIN).getMessage(player.getLocale()));```
But it doesn't show anything, i don't quite understeand how I can get localized text into a base component :/

**Comment by twizmwazin** [comment]
@Pablete1234 Just make a TextComponent with the legacy text as the constructor paramater

**Comment by Pablete1234** [comment]
@twizmwazin Could you give me some examples?

**Comment by twizmwazin** [comment]
new TextComponent(ChatColor.RED + "This text is red " + ChatColor.BLUE + "but this text is blue.")

**Comment by Pablete1234** [comment]
So you mean I should use a TextComponent instead of a BasicComponent?

**Comment by twizmwazin** [comment]
TextComponent extends BaseComponent

**Comment by Pablete1234** [comment]
@twizmwazin opinions?

**Comment by Pablete1234** [comment]
@iEli2tyree011 Said in #750 that I should add titles, I added them, anything else I should add?

File: src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java [hunk 2]
```diff
@@ -62,16 +63,23 @@ public void run() {
...
-                    if (Settings.getSettingByName("Sounds") != null && Settings.getSettingByName("Sounds").getValueByPlayer(player).getValue().equalsIgnoreCase("on")) {
+                    if (Settings.getSettingByName("Sounds") != null && Settings.getSettingByName("Sounds").getValueByPlayer(player).getValue().equalsIgnoreCase("on") && !Teams.getTeamByPlayer(player).get().isObserver()) {
                         player.playSound(player.getLocation(), Sound.NOTE_PLING, 1, 2);
                     }
+                    if (!Teams.getTeamByPlayer(player).get().isObserver()) {
+                        String title = new LocalizedChatMessage(ChatConstant.UI_MATCH_START_TITLE).getMessage(player.getLocale());
+                        player.showTitle(new TextComponent(ChatColor.GREEN + title), new TextComponent(""), 0, 10, 10);
+                    }
                 }
             }
```
Comment (Alfalik) [inline src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java]: You can add the 3, 2 ,1 countdown with the following code:

```java
for (Player player : Bukkit.getOnlinePlayers()) {
    if (!TeamUtils.getTeamByPlayer(player).isObserver()) {
        if (time == 0) {
            player.showTitle(ChatColor.GREEN + "Go!"); // You already coded this, just move it.
        } else if (((time % 20) == 0) && (time <= 60)) {
            player.showTitle(new TexrComponent(ChatColor.YELLOW + (time / 20)), new TextComponent(""), 0, 10, 10);
        }
    }
}
```

Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java]: The countdown is already coded with the 3,2 and 1 sounds, and the "Go!" message is localized with the time == 0
also i wouldn't "for all players -> if count =0" it makes more sense "if count = 0. for all players"

Comment (Alfalik) [inline src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java]: Uh, nevermind. Didn't see it, my bad.

**Comment by TiTo-418** [comment]
TESTED WORKS PERFECT PLEASE MERGE IT'S AWSOMELY PERFECT

File: src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java [hunk 3]
```diff
@@ -31,24 +31,36 @@ public void unload() {
...
-                ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.DARK_PURPLE + "# # {0}" + ChatColor.DARK_PURPLE + " # #", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getColor() + (event.getTeam().get()).getName())));
+                ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage("{0}", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getColor() + (event.getTeam().get()).getName() + ChatColor.WHITE)));
             } else {
-                ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.DARK_PURPLE + "# # {0}" + ChatColor.DARK_PURPLE + " # #", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getCompleteName())));
+                ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage("{0}", new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getCompleteName() + ChatColor.WHITE)));
+            }
+            for (Player player : Bukkit.getOnlinePlayers()) {
+                if (Teams.getTeamByPlayer(player).get() == event.getTeam().get()) {
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java]: Don't assume they are on a team

File: src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java [hunk 4]
```diff
@@ -31,24 +31,36 @@ public void unload() {
...
+                    player.sendMessage(new UnlocalizedChatMessage(ChatColor.GREEN + "{0}", new LocalizedChatMessage(ChatConstant.UI_MATCH_TEAM_WIN)).getMessage(player.getLocale()));
+                    String title = new LocalizedChatMessage(ChatConstant.UI_MATCH_WIN, event.getTeam().get().getColor() + (event.getTeam().get()).getName() + ChatColor.WHITE).getMessage(player.getLocale());
+                    String subtitle = new LocalizedChatMessage(ChatConstant.UI_MATCH_TEAM_WIN).getMessage(player.getLocale());
+                    player.showTitle(new TextComponent(title),new TextComponent(ChatColor.GREEN + subtitle), 0, 40, 30);
+                } else if (!Teams.getTeamByPlayer(player).get().isObserver()) {
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/module/modules/match/MatchModule.java]: same thing here

**Pull request closed without merging**